### ISSUE TITLE
Display badge-related criteria results

### ISSUE THREAD

**Issue by orviz** [EOSC-synergy/sqaaas-web#153]
Not all the criteria is relevant for the badge issuance process. For instance, although the `QC.Met` is theoretically part of the bronze category for software, there are no *MUST* subcriteria defined for such criterion, and thus, the criterion is not considered when issuing badges. The criteria that matters for this issue are the ones actually considered in the badge issuance process. In the previous example they are `QC.Lic`, `QC.Acc` and `QC.Doc` (all of them contain *MUST* subcriteria).

The purpose of this issue is to make available in the final view the set of criteria (**not subcriteria**) being fulfilled and not fulfilled for each badge category. This could be presented as a sort of table having each row the criteria (showing whether they are fulfilled or not fulfilled) that are considered for each type of badge category (gold, silver, bronze). The latter represented by (gold, silver, bronze) icons.

This information shall be displayed next to the badge data (image, verfication URL, etc.)

**Comment by orviz** [comment]
A first approach is available in the [API spec release/2.5.0](https://github.com/EOSC-synergy/sqaaas-api-spec/tree/release/2.5.0). It adds the `badge:<type>:<criteria>` property where each class of badge contains an object with the `to_fulfill`, `missing` and `fulfilled` arrays that contain the list of relevant criterion IDs, such as follows:

```json
{
  "report": { },
  "badge": {
    "software": {
      "criteria": {
        "bronze": {
          "to_fulfill": ["criterion_1", "criterion_4"],
          "missing": [],
          "fulfilled": []
        },
        "silver": {
          "to_fulfill": [],
          "missing": [],
          "fulfilled": []
        },
        "gold": {
          "to_fulfill": [],
          "missing": [],
          "fulfilled": []
        },
      },
      "data": {      },
      "share": "<html>..</html>",
      "verification_url": "http://example.com"
    },
  }
}

**Comment by orviz** [comment]
Addressed in https://github.com/EOSC-synergy/sqaaas-web/pull/203

**Comment by orviz** [comment]
Released under [1.6.0](https://github.com/EOSC-synergy/sqaaas-web/releases/tag/1.6.0)
